db/python/tables/family.py

```diff
@@ -29,16 +29,33 @@ async def get_projects_by_family_ids(self, family_ids: List[int]) -> Set[Project
             )
         return projects
 
-    async def get_families(self, project: int = None) -> List[Family]:
+    async def get_families(
+        self, project: int = None, participant_ids: List[int] = None
+    ) -> List[Family]:
         """Get all families for some project"""
-        _query = """\
-SELECT id, external_id, description, coded_phenotype, project
-FROM family
-WHERE project = :project"""
+        _query = """
+            SELECT id, external_id, description, coded_phenotype, project
+            FROM family
+        """
 
-        rows = await self.connection.fetch_all(
-            _query, {'project': project or self.project}
-        )
+        values: Dict[str, Any] = {'project': project or self.project}
+        where: List[str] = []
+
+        if participant_ids:
```

Comment:
This is returning multiple families for multiple individuals specified within the family. I suspect this will be cleaner to write two different queries for each case (no participants OR participants).